.github/actions/bpftrace/scripts/check-encryption-leaks.bt

```diff
@@ -223,7 +232,7 @@ kprobe:br_forward
         if (!$tcph->rst) {
           $time = strftime("%H:%M:%S:%f", nsecs);
 
-          printf("[%s] [%p] %s:%d -> %s:%d (len: %d, proto: %d, encap: %d, iif: %d, netns: %x, srcPod: %d(%d), dstPod: %d(%d), proxy: %d)\n",
+          printf("[%s] [%p] %s:%d -> %s:%d (len: %d, proto: %d, encap: %d, iif: %d, netns: %x, srcPod: %d(%d), dstPod: %d(%d), proxy: %d(%d))\n",
```

Comment:
even if I reviewed this hours ago, I forget what's inside the parenthesis. Maybe
```
srcPod: %d(internal: %d)
```
?

Reply:
Agree, I just hoped the output would stay within 1-line... but at least as you mention it will be more readable.
Will do similar update also for the `encap: %d(%d)` and `proxy: %d(%d)`.